app/assets/src/components/views/discovery/discovery_sidebar.scss

```diff
@@ -120,7 +120,7 @@
     height: 12px;
     margin-left: 5px;
     margin-right: 7px;
-    padding: 0 5px;
+    padding-left: 1px;
```

Comment:
I threw in this change to reduce padding on the bars because it was giving it a min width of 10px which was too off scale

![Screen Shot 2019-04-01 at 10 25 35 AM](https://user-images.githubusercontent.com/5652739/55347070-c205c200-5468-11e9-85e3-f2f73c3eeda9.png)

Reduced:
![Screen Shot 2019-04-01 at 10 26 16 AM](https://user-images.githubusercontent.com/5652739/55347076-c631df80-5468-11e9-94a3-4551f66a0d4f.png)